examples/typescript-reactflux/gulpfile.js

```diff
@@ -0,0 +1,332 @@
+	    ['_', 'isNull'],
+	    ['_', 'isDefined']
+	],
+	'class-name': true,
+	'comment-format': [false,
+	    'check-space',
+	    'check-lowercase'
+	],
+	'curly': true,
+	'eofline': true,
+	'forin': true,
+	'indent': [true, 4],
+	'interface-name': false,
+	'jsdoc-format': false,
+	'label-position': true,
+	'label-undefined': true,
+	'max-line-length': [true, 140],
+	'member-ordering': [false,
+	    'public-before-private',
+	    'static-before-instance',
+	    'variables-before-functions'
+	],
+	'no-arg': true,
+	'no-bitwise': false,
+	'no-console': [true,
+	    'debug',
+	    'info-false',
+	    'time',
+	    'timeEnd',
+	    'trace'
+	],
+	'no-construct': true,
+	'no-constructor-vars': true,
+	'no-debugger': true,
+	'no-duplicate-key': true,
+	'no-duplicate-variable': true,
+	'no-empty': false,
+	'no-eval': true,
+	'no-string-literal': true,
+	'no-switch-case-fall-through': true,
+	'no-trailing-comma': true,
+	'no-trailing-whitespace': false,
+	'no-unused-expression': false,
+	'no-unused-variable': true,
+	'no-unreachable': true,
+	'no-use-before-declare': true,
+	'no-var-requires': false,
+	'one-line': [false,
+	    'check-open-brace',
+	    'check-catch',
+	    'check-else',
+	    'check-whitespace'
+	],
+	'quotemark': [false, 'double'],
+	'radix': true,
+	'semicolon': true,
+	'triple-equals': [false, 'allow-null-check'],
+	'typedef': [true,
+	    'callSignature',
+	    'indexSignature',
+	    'parameter',
+	    'propertySignature',
+	    'variableDeclarator'
+	],
+	'typedef-whitespace': [true,
+	    ['callSignature', 'noSpace'],
+	    ['catchClause', 'noSpace'],
+	    ['indexSignature', 'space']
+	],
+	'use-strict': [false,
+	    'check-module',
+	    'check-function'
+	],
+	'variable-name': false,
+	'whitespace': [false,
+	    'check-branch',
+	    'check-decl',
+	    'check-operator',
+	    'check-separator',
+	    'check-type'
+	]
+      }
+    }
+  };
+
+  return gulp.src(config.build + '/src/**/*.ts')
+    .pipe(tslint(options))
+    .pipe(tslint.report('prose'));
+});
+
+/**
+  The 'debug' task does not include:
+  jsx, tslint
+ */
+gulp.task('debug', function(callback) {
+  return runSequence(
+	  'clean',
+	  ['typings','src'],
+	  'ts',
+	  'bundle',
+	  callback);
+});
+
+
+gulp.task('release', function(callback) {
+  return runSequence(
+	  'clean',
+	  'tsd',
+	  ['typings','jsx'],
+	  'tslint',
+	  'ts',
+	  'bundle',
+	  'minify',
+	  callback);
+});
+
+gulp.task('lint', function(callback) {
+  return runSequence(
+	  'clean',
+	  'tsd',
+	  ['typings','jsx'],
+	  'tslint',
+	  callback);
+});
+
+gulp.task('help', function(callback) {
+  gutil.log('');
+  gutil.log('USAGE:');
+  gutil.log('Open web/index.html in browser.');
+  gutil.log('');
+  gutil.log('TASKS:');
+  gutil.log('debug   - builds web/js/bundle.js with runtime JSX renderer.');
+  gutil.log('release - builds web/js/bundle.min.js with expanded JSX.');
+  gutil.log('');
+});
+
+gulp.task('build', ['release']);
+
+gulp.task('default', ['release']);
```

Comment:
It (and many other code in this PR) uses two space indentation, we require tabs :)